content/applications/websites/website/configuration/multi_website.rst

```diff
@@ -2,316 +2,164 @@
+target audience.
+
+Each website can be designed and configured independently with its own :doc:`domain name
+<../../../../administration/maintain/domain_names>`, theme, pages,
+menus, :doc:`languages <translate>`, :doc:`products <../../ecommerce/managing_products/products>`,
+assigned sales team, etc. They can also :ref:`share contents and pages
+<multi-website/website_content>`.
 
-Odoo’s Multi-Websites opens up broad possibilities of diversification and
-customer segmentation for your business. A multiplied audience and
-boosted revenue are now just a few clicks away!
-
-Each website can work in a fully independent way, with its theme,
-branding, domain name, header & footer, pages, languages, products, blog
-posts, forum, slides, events, live chat channels, etc. Let’s go for a
-tour!
-
-Setup
-=====
-
-To create a new website, go to :menuselection:`Website --> Configuration --> Settings`.
-The button, *Create a new website*, lays in the first section.
-
-
-
-.. image:: multi_website/multi_website05.png
-  :align: center
+.. tip::
+   Duplicate content (i.e., pages and content shared between multiple websites) can have a negative
+   impact on :doc:`../pages/seo`.
 
-In the upcoming prompt, set a name for your new website and a specific
-domain name. Leave empty to publish the new website under the default
-domain of your Odoo database. You can later set some country groups to
-redirect visitors to it using Geo IP.
+Website creation
+================
 
-.. image:: multi_website/multi_website01.png
-  :align: center
+To create a new website, proceed as follows:
 
-Then, select a theme. This new website might have an entirely different
-purpose or audience than the first one. So feel free to go for a
-different theme!
+#. Go to :menuselection:`Website --> Configuration --> Settings`.
+#. Click :guilabel:`+ New Website`.
 
-Once the theme is selected, you can start to build the homepage of your
-website. Follow the purple drops; they will help you in the first steps.
+   .. image:: multi_website/create-website.png
+      :alt: New website button
 
-.. image:: multi_website/multi_website08.png
-  :align: center
+#. Specify the :guilabel:`Website Name` and :guilabel:`Website domain`. Each website must be
+   published under their own :doc:`domain <../../../../administration/maintain/domain_names>`.
+#. Adapt the :guilabel:`Company name`, :guilabel:`Languages` and :guilabel:`Default language`
+   if needed.
+#. Click the :guilabel:`Create` button.
 
+You can then start building your new website.
 
 .. note::
-   If you run Odoo Online, don’t forget to redirect any new domain name to your Odoo database
-   (``CNAME``) and to authorize it Odoo-side. See :ref:`domain-name/existing`.
-
-Create the menu
-===============
+   By default, all website-related apps/modules that you have installed (e.g. **eCommerce**,
+   **Forum**, **Blog**, etc.) and their related pages (e.g., **Forum**) are also available on the
```

Comment:
why the `(e.g., **Forum**)`? Don't they all have a related page? I first thought that you meant that the Forum pages are the same on both websites. Maybe it is, actually? ^^

Reply:
It was just an example to illustrate that if you have installed the Forum app, by default, the Forum page will be there on the new website when you create it. 
And you can restrict a forum to a specific website (using the Website field) and have different forums for different websites. 
Does this answer your question?

Reply:
yes, but then I think you can just leave "and their related pages". Up to you :)

Reply:
Done ;)